onchain/rollups/test/foundry/inputs/InputBox.t.sol

```diff
@@ -158,28 +158,45 @@ contract InputBoxTest is Test {
         assertEq(inputBox.getNumberOfInputs(_dapp), 0);
     }
 
+    function getMaxInputPayloadLength() internal pure returns (uint256) {
+        // an input blob instance
+        bytes memory blob = LibInput.encodeEvmInput(
```

Comment:
Here, you can call a contract that has a function with the exact same signature of `encodeEvmInput` which calls this function from `LibInput`. It's perfectly ok to call an external function that expects `calldata` with `memory`.